### ISSUE TITLE
[bug] [core] Image files in plugin templates get corrupted, when copied into the app.

### ISSUE THREAD

**Issue by kosmas58** [Compass/compass#39]
# Issue

I've images included in the templates of a compass plugin. In manifest.rb I've written something
like:

```
image 'jquery.ui/dynatree.base/cbChecked.png'
```

When installing the template, the template is installed as described in manifest.rb, but the images are corrupt.
## Possible reason

I think the issue occurs incompass/lib/actions.rb: 

```
    compass/lib/actions.rb: 

   def copy(from, to, options = nil)
      ...
      contents = File.new(from).read
      write_file to, contents, options
   end

   def write_file(file_name, contents, options = nil)
      ...
      file.write(contents)
      ...   
    end
```

This seems to break files with binary content.

**Comment by chriseppstein** [comment]
grrrrr. Yes, that should write the file in binary mode...

**Comment by chriseppstein** [comment]
Fixed in v0.8.10

**Comment by chriseppstein** [comment]
I received a report that this bug still exists. Can you confirm or deny?
